### PR TITLE
set up webpack config

### PR DESCRIPTION
**Pull request by aliceshen227**

### CODE REVIEW

File: webpack.config.js [hunk 1]
```diff
@@ -0,0 +1,24 @@
+
```
Comment (ikarasz) [inline webpack.config.js]: please remove empty line

File: webpack.config.js [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+      filename: 'bundle.js',
+  },
+  module: {
+      loaders: [
+          {
+              test: /\.js$/,
+              include: path.resolve(__dirname, 'src'),
+              loader: 'babel-loader',
+              query: {
+                  presets: ['react']
```
Comment (ikarasz) [inline webpack.config.js]: please add es2015 as preset

File: webpack.config.js [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+              }
+          },
+          {
+            test:/\.scss$/,
```
Comment (ikarasz) [inline webpack.config.js]: please add space after colon

File: webpack.config.js [hunk 4]
```diff
@@ -0,0 +1,24 @@
...
+            use:['style-loader','css-loader','sass-loader']
```
Comment (ikarasz) [inline webpack.config.js]: please add space after comma

**Review by ikarasz — CHANGES_REQUESTED** [review]
please pay more attention to the indentation (2spaces)

**Comment by aliceshen227** [comment]
I changed the style and added preset.

File: webpack.config.js [hunk 5]
```diff
@@ -0,0 +1,23 @@
...
+    filename: 'bundle.js', 
+  }, 
+  module: {
+    loaders: [
+      {
+        test: /\.js$/,
+        include: path.resolve(__dirname, 'src'),
+        loader: 'babel-loader',
+        query: {
+          presets: ['react', 'env']
```
Comment (ikarasz) [inline webpack.config.js:14]: missing es2015 preset

File: webpack.config.js [hunk 6]
```diff
@@ -0,0 +1,23 @@
...
+        }
+      },
+      {
+        test:/\.scss$/,
```
Comment (ikarasz) [inline webpack.config.js]: missing space after colon

File: webpack.config.js [hunk 7]
```diff
@@ -0,0 +1,23 @@
...
+        use:['style-loader', 'css-loader', 'sass-loader']
```
Comment (ikarasz) [inline webpack.config.js]: missing space after colon

**Review by ikarasz — CHANGES_REQUESTED** [review]
everything what is used here should be added also to the package.json

**Review by ikarasz — APPROVED** [review]

**Pull request merged**